src/media-queries/mediaqueries.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
when you start a project on an html doc in vscode, type: `doc` and hit tab. it will fill out all this html/head stuff that is needed.